docs/client-libraries-clients.md

```diff
@@ -51,4 +51,36 @@ The following example creates a Python client using multiple advertised listener
 
   </TabItem>
 </Tabs>
+````
+
+## Set memory limits
+
+You can uses memory limits param to control client overall memory usage,
+the producers and consumers under this client will compete for the memory assigned. See [PIP 74: Pulsar client memory limits](https://github.com/apache/pulsar/wiki/PIP-74%3A-Pulsar-client-memory-limits)
```

Comment:
```suggestion
the producers and consumers under this client will compete for the memory assigned. For implementation details, see [PIP 74: Pulsar client memory limits](https://github.com/apache/pulsar/wiki/PIP-74%3A-Pulsar-client-memory-limits).
```

Reply:
Do we have any considerations/limitations that need to remind users?

Reply:
I think nothing.